web_sdk/golden_comparator/pubspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: golden_comparator
```

Comment:
Does this need to be a separate package? Can we put it in the common test utilities?

Reply:
done.